app/src/main/java/com/strv/movies/ui/moviedetail/MovieDetailScreen.kt

```diff
@@ -82,29 +84,49 @@ fun MovieTrailerPlayer(videoId: String, progressSeconds: MutableState<Float>) {
 
 @Composable
 fun MoviePoster(movie: MovieDetail) {
-    AsyncImage(
-        model = "https://image.tmdb.org/t/p/w500${movie.posterPath}",
-        contentDescription = stringResource(id = R.string.movie_image),
-        modifier = Modifier
-            .padding(top = 16.dp)
-            .size(120.dp)
-    )
+    Column {
+
+        AsyncImage(
+            model = "https://image.tmdb.org/t/p/w500${movie.posterPath}",
+            contentDescription = stringResource(id = R.string.movie_image),
+            modifier = Modifier
+                .padding(top = 16.dp)
+                .size(120.dp),
+        )
+        Row {
+            Text(
+                movie.title,
+                fontWeight = FontWeight.Bold,
+                textAlign = TextAlign.Center,
+                fontSize = 10.sp,
+                color = Color.Red
+            )
+        }
+    }
 }
 
 @Composable
 fun MovieInfo(movie: MovieDetail) {
     Column {
         Text(
             movie.title,
-            modifier = Modifier.padding(top = 16.dp, end = 16.dp),
+            modifier = Modifier
+                .padding(start = 10.dp, top = 16.dp, end = 10.dp),
             fontWeight = FontWeight.Bold,
             fontSize = 20.sp
         )
-        Text(movie.releaseDate, modifier = Modifier.padding(top = 8.dp))
+        Text(
+            movie.releaseDate,
+            modifier = Modifier
+                .padding(start = 10.dp, top = 8.dp)
+        )
         Text(
             movie.overview,
-            modifier = Modifier.padding(top = 8.dp, end = 16.dp),
-            textAlign = TextAlign.Justify
+            modifier = Modifier
+                .padding(start = 10.dp, top = 8.dp, end = 10.dp)
+                .size(240.dp),
```

Comment:
Setting the fixed size here is not ideal, because if you have phone with different size the `MovieInfo` + `MoviePoster` wouldn't fill the screen or will overflow it. In this case you can f.e. set fixed size to MoviePoster and let the MovieInfo to fill the rest of the screen. It could be done f.e. by `weigh` modifier like this:
```
Row {
            MovieInfo(
                movie = movie,
                modifier = Modifier.weight(1f)
            )
            MoviePoster(movie = movie)
        }
```
To be able to do that you have to define the modifier parameter for the MovieInfo.

Reply:
done